libsolidity/ast/AST.h

```diff
@@ -1463,19 +1463,26 @@ class InlineAssembly: public Statement
 		SourceLocation const& _location,
 		ASTPointer<ASTString> const& _docString,
 		yul::Dialect const& _dialect,
+		ASTPointer<std::vector<ASTPointer<ASTString>>> _flags,
 		std::shared_ptr<yul::Block> _operations
 	):
-		Statement(_id, _location, _docString), m_dialect(_dialect), m_operations(std::move(_operations)) {}
+	Statement(_id, _location, _docString),
+	m_dialect(_dialect),
+	m_flags(move(_flags)),
+	m_operations(std::move(_operations))
+	{}
 	void accept(ASTVisitor& _visitor) override;
 	void accept(ASTConstVisitor& _visitor) const override;
 
 	yul::Dialect const& dialect() const { return m_dialect; }
 	yul::Block const& operations() const { return *m_operations; }
+	auto const& flags() const { return m_flags; }
```

Comment:
I don't want to look into the private details of a class to find out its interface.

Reply:
Haha, it took me a full minute to understand what you mean - I don't even see this stuff :-). But fair enough, you're right :-).